app/src/main/java/org/openobservatory/ooniprobe/domain/GenerateAutoRunServiceSuite.java

```diff
@@ -46,8 +46,6 @@ public AbstractSuite generate(
                     )
             );
             OONIContext ooniContext = session.newContextWithTimeout(30);
-            session.maybeUpdateResources(ooniContext);
-
```

Comment:
The `maybeUpdateResources` API call has been a stub for quite some time. Removing it is harmless.